### ISSUE TITLE
Adding useful definitions

### ISSUE THREAD

**Issue by nicholasamorim** [marshmallow-code/apispec#401]
Hi, thanks for this great project.

Maybe I'm missing something but is there any way we can document individual fields? I'm using APISpec couple with Marshmallow and while the the response objects are rendered just fine, I'm wondering if there's any way of adding meaningful descriptions to each field.

**Comment by lafrech** [comment]
Yes.

https://apispec.readthedocs.io/en/stable/api_core.html#apispec.core.Components.schema

If you are using apispec.ext.marshmallow, you can pass fields’ metadata as additional keyword arguments.

For example, to add enum and description to your field:

```py
status = fields.String(
    required=True,
    enum=['open', 'closed'],
    description='Status (open or closed)',
)
```

(We could reword this as "If you are using `MarshmallowPlugin`...")

**Comment by nicholasamorim** [comment]
Oh, perfect. I knew was I was missing something. Thanks!
